spec/factories/users.rb

```diff
@@ -53,7 +53,7 @@
   end
 
   sequence(:strong_password) do |n|
-    [Faker::Name.first_name, Faker::Name.last_name, Faker::Internet.domain_word, n].join("-")
```

Comment:
This probably couldn't have failed the new validations, but I'd still prefer a programmatically impossible failure.